docs/cli_zh.md

```diff
@@ -1,3 +1,79 @@
 # Fling 的 CLI 使用
 
-TBD
+在这份文档中，我们将介绍 Fling 中命令行界面（CLI）的使用方式。
+
+## 版本
+
+您可以使用如下命令：
+
+```shell
+fling -v
+```
+
+或者：
+
+```shell
+fling --version
+```
+
+以列出您当前已安装的 Fling 版本。同时也可以检查 Fling 是否成功安装。
+
+## fling run 命令
+
+您可以使用 `fling run` 来根据配置文件启动一次实验。以下是一个用法示例：
+
+```shell
+fling run -c argzoo/mnist/mnist_fedper_cnn_toy_config.py -p personalized_model_pipeline -s 1,2,3
+```
+
+以下是关于每个参数的含义解释。
+
+- -c / --config：您想要使用的配置文件。
+- -p / --pipeline：您想要执行的 pipeline。此 pipeline 应包含在 `fling.pipeline` 中。
+- -s / --seed：您想要运行的 seeds。它可以是一个整数，也可以是类似示例中的多个整数。默认情况下，该 seed 被设定为0。
```

Comment:
Explain the ``seed`` here briefly. You can refer to: https://en.wikipedia.org/wiki/Random_seed. Also add this for English doc.